packages/frontend/amp/lib/find-adslots.ts

```diff
@@ -139,3 +140,14 @@ export const findAdSlots = (elements: CAPIElement[]): number[] => {
     }
     return adSlots;
 };
+
+// Returns index of items to place ads *after*
+export const findBlockAdSlots = (blocks: any[]): number[] => {
+    const maxAds = 8;
+
+    return blocks
```

Comment:
This is a little bit dense to parse. Can we reduce the terseness or add comments so it's easier to comprehend?

Reply:
Yeah, definitely. I've added a comment.